creed/qc_comp.py

```diff
@@ -0,0 +1,505 @@
+import copy
+import matplotlib.pyplot as plt
+import os
+import numpy as np
+import pandas as pd
+from requests.exceptions import HTTPError
+from scipy.ndimage import gaussian_filter
+import seaborn as sns
+from shutil import rmtree
+from skimage.io import imsave
+
+from creed.mibitracker_utils import MibiTrackerError
+from creed.mibitracker_utils import MibiRequests
+
+import ark.settings as settings
+import ark.utils.io_utils as io_utils
+import ark.utils.load_utils as load_utils
+import ark.utils.misc_utils as misc_utils
```

Comment:
I feel like we could avoid these ark imports for a future version.  Either via reimplementing the requisite tools in the creed repo, or consolidating our io related code into another separate repo/project (which sounds silly, but in almost every mibi-related project, I usually end up installing the entirety of ark for these io/loading utils anyways)

Reply:
Agreed. This would be a nice starter project for our new hires